packages/react-components/react-infobutton/Spec.md

```diff
@@ -1,63 +1,143 @@
 # @fluentui/react-infobutton Spec
 
+Convergence epic issue: [#25062](https://github.com/microsoft/fluentui/issues/25062)
+
 ## Background
 
-_Description and use cases of this component_
+An InfoButton provides a way for users to get more information about a particular UI element. It is a button that displays an icon and when clicked a Popover displays the additional information.
+
+Because the Popover isn't always visible, it should not contain information that people must know in order to complete the field.
+
+### Anatomy
+
+![Anatomy](./etc/images/anatomy.png)
 
 ## Prior Art
 
-_Include background research done for this component_
+| Name                                                | library                                                                               | Notes                                                                                                    |
+| --------------------------------------------------- | ------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| IconButtonTooltip                                   | [Carbon Design System](https://www.carbondesignsystem.com/components/tooltip/usage)   | This is an example within the tooltip component.                                                         |
+| Tooltip with card appearance and general tooltip    | [EverGreen](https://evergreen.segment.com/components/tooltip)                         | This are examples within the tooltip component. This library doesn't use a button, instead just an icon. |
+| Input and guidance to add info button               | [Salesforce](https://www.lightningdesignsystem.com/components/input/)                 | This component lives within Input and shows how to build an info button instead.                         |
+| TextField with example on how to add an info button | [v8](https://developer.microsoft.com/en-us/fluentui#/controls/web/textfield)          | This is just an example and on how to implement it.                                                      |
+| ContextualHelp                                      | [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/ContextualHelp.html) | It's a full component and uses a Popover instead of tooltip.                                             |
+
+#### Comparison with v0 and v8
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- v0 does not have an InfoButton component.
+- v8 does not have an InfoButton component, but does have a TextField component that has an example of how to add an info button to a TextField, see Open UI section above for more details.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```jsx
+<InfoButton content="This is some additional information." />
```

Comment:
One of the images shows a link in the popover. Would be nice to have a code sample showing you can use JSX in the `content` prop to make it clear we `InfoButton` supports that.